profile/README.md

```diff
@@ -5,75 +5,45 @@ OrbitDB uses [IPFS](https://ipfs.io/) as its data storage and [IPFS Pubsub](http
 
-And then, there are a whole host of modules. It should be noted that **all** of these are JavaScript modules on npm.
-
-- **[crdts](https://github.com/orbitdb/crdts)** - A library of [Conflict-Free Replicated Data Types](https://en.wikipedia.org/wiki/Conflict-free_replicated_data_type) for JavaScript. Used in [orbit-db-counterstore](https://github.com/orbitdb/orbit-db-counterstore); mostly just one function is used.
-- **[orbit-db-cache](https://github.com/orbitdb/orbit-db-cache)** - Actually a persistency layer, not a cache. Saves the latest HEAD of each db locally into LevelDB. Doesn't store a huge amount of data, just the amount of DBs you have. For all of the dbs you have locally, you have a hash for each.
-- **[orbit-db-cli](https://github.com/orbitdb/orbit-db-cli)** - CLI for OrbitDB.
-- **[orbit-db-counterstore](https://github.com/orbitdb/orbit-db-counterstore)** - Datastore in OrbitDB. Uses CRDTs, inherits from [orbit-db-store](https://github.com/orbitdb/orbit-db-store).
-- **[orbit-db-docstore](https://github.com/orbitdb/orbit-db-docstore)** - Another OrbitDB store which allows you to index a blob.
-- **[orbit-db-eventstore](https://github.com/orbitdb/orbit-db-eventstore)** - Another data model type. Like [ipfs-log](https://github.com/orbitdb/ipfs-log). Wraps into a store interface.
-- **[orbit-db-feedstore](https://github.com/orbitdb/orbit-db-feedstore)** - Database type in OrbitDB. This is exactly like eventlog, but you can delete events from the log.
-- **[orbit-db-keystore](https://github.com/orbitdb/orbit-db-keystore)** - Keystore is actually a module that stores the keys locally. You can generate keys, it persists them in the data folder, you can import and export keys - it is basically a key manager.
-- **[orbit-db-kvstore](https://github.com/orbitdb/orbit-db-kvstore)** - KeyValue store. Another DB type in OrbitDB. This may occasionally be unclear in the code, where `kvstore` is called `keyvalue` store.
-- **[orbit-db-pubsub](https://github.com/orbitdb/orbit-db-pubsub)** - Message propagation module for OrbitDB. A layer between OrbitDB and IPFS Pubsub.
-- **[orbit-db-store](https://github.com/orbitdb/orbit-db-store)** - Base class for OrbitDB data stores. A Base Class defines the common interface for all databases.
-- **[orbit-db-test-utils](https://github.com/orbitdb/orbit-db-test-utils)** - Reusable utilities for automated testing across the OrbitDB stack
-- **[orbit-db-benchmark-runner](https://github.com/orbitdb/benchmark-runner)** - Benchmark runner for compatible orbit-db repositories.
-- **[orbit-db-storage-adapter](https://github.com/orbitdb/orbit-db-storage-adapter)** - Adapter for any leveldown-compliant store
-- **[orbit-db-io](https://github.com/orbitdb/orbit-db-io)** - IO helper module used within OrbitDB
+**[DID Identity Provider](https://github.com/orbitdb/orbitdb-identity-provider-did)** - A Decentralized IDentitifer identity provider for OrbitDB.
 
 ### Multilingual Implementations
```

Comment:
For this we could a) call it "Implementations" and b) point to the Berty project ("A Go implementation is developed and maintained by the [Berty](https://github.com/berty) project at [berty/go-orbit-db](https://github.com/berty/go-orbit-db").

I don't think orbit-db-http-api is up to date nor maintained, not sure about the status of https://github.com/orbitdb/py-orbit-db-http-client. We can add these back if they are maintained and updated to 1.0.